### ISSUE TITLE
Training Dataset question

### ISSUE THREAD

**Issue by kklucy11** [StanfordMIMI/DDM2#34]
Hi, thank you for this amazing paper. I wanted to ask you very few questions to elaborate in very detail.
Should the data used for training during the training process be 1 4D data?
Can a model trained on only one 4D data be used on other people's 4D data?

**Comment by tiangexiang** [comment]
Yes. DDM^2 is a **self-supervised optimization-based** denoising algorithm. That means the training data needs to be the testing data itself. With this being said, we require one single 4D MRI acquisition (more volumes are better) as the input. After training, the trained model is able to (and is only able to) denoise the **same** 4D MRI acquisition.
